tests/system/providers/amazon/aws/example_batch.py

```diff
@@ -209,6 +220,9 @@ def delete_job_queue(job_queue_name):
     # [END howto_sensor_batch]
     wait_for_batch_job.poke_interval = 10
 
+    # Only describe the job if the task above failed, to help diagnose
```

Comment:
```suggestion
    # Only describe the job if the tasks above failed, to help diagnose
```

Also maybe move the TriggerRule setting to here? Otherwise this comment doesn't make much sense (or move the comment up to the task definition where the TriggerRule is specified now).